src/App.tsx

```diff
@@ -20,17 +25,127 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    imgWidth: 130,
+    animationDur: 1000,
+    infinite: false,
+
+  };
+
+  inputHandler = (event: React.FocusEvent<HTMLInputElement>) => {
+    switch (event.currentTarget.name) {
+      case 'itemWidth': {
+        if (+event.currentTarget.value > 0) {
+          this.setState({ imgWidth: +event.currentTarget.value });
+        } else {
+          this.setState({ imgWidth: 130 });
+        }
+
+        return;
+      }
+
+      case 'frameSize': {
+        if (+event.currentTarget.value
+          && +event.currentTarget.value <= this.state.images.length) {
+          this.setState({ frameSize: +event.currentTarget.value });
+        } else {
+          this.setState({ frameSize: 3 });
+        }
+
+        return;
+      }
+
+      case 'step': {
+        if (+event.currentTarget.value > 0) {
+          this.setState({ step: +event.currentTarget.value });
+        } else {
+          this.setState({ step: 3 });
+        }
+
+        return;
+      }
+
+      case 'animationDuration': {
+        if (+event.currentTarget.value > 500) {
+          this.setState({ animationDur: +event.currentTarget.value });
+        } else {
+          this.setState({ animationDur: 1000 });
+        }
+
+        break;
+      }
+
+      default:
+    }
   };
 
   render() {
-    const { images } = this.state;
+    const {
+      images,
+      step,
+      frameSize,
+      imgWidth,
+      animationDur,
+      infinite,
+    } = this.state;
 
     return (
       <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
-
-        <Carousel />
+        <h1 data-cy="title">{`Carousel with ${images.length} images`}</h1>
+        <Carousel
+          images={images}
+          step={step}
+          frameSize={frameSize}
+          itemWidth={imgWidth}
+          animationDuration={animationDur}
+          infinite={infinite}
+        />
+        <div className="inputs-container">
+          <label htmlFor="itemWidth">
+            Image size:
+            <input
+              type="number"
+              name="itemWidth"
+              id="itemWidth"
+              min={50}
+              placeholder="130"
+              onBlur={this.inputHandler}
```

Comment:
state of inputs should be updates onChange event
```suggestion
              onChange={this.inputHandler}
```
![carousel](https://github.com/mate-academy/react_carousel/assets/33029864/4fa399f6-5dbf-4590-b665-df0ea2b003f0)